api/wallet.go

```diff
@@ -65,6 +65,13 @@ type (
 		AllSeeds           []string `json:"allseeds"`
 	}
 
+	// WalletSweepPOST contains the coins and funds discovered by a call to
+	// /wallet/sweep.
```

Comment:
comment should make it a little more clear that this is the return value, not the input value.

Though iirc by convention inputs are always path parameters